testsuite/features/secondary/srv_virtual_host_manager.feature

```diff
@@ -24,6 +24,7 @@ Feature: Virtual host manager web UI
 
   Scenario: Create Virtual Host
     When I follow the left menu "Systems > Virtual Host Managers"
+    And I copy vcenter configuration file on server
```

Comment:
VMware writes it like this: `vCenter`

https://www.vmware.com/products/vcenter-server.html

`onto the server`